web/src/stream_data.js

```diff
@@ -860,3 +862,21 @@ export function get_options_for_dropdown_widget() {
         }))
         .sort((a, b) => util.strcmp(a.name.toLowerCase(), b.name.toLowerCase()));
 }
+
+export function user_is_topic_participant(stream_id, topic, current_user_id) {
+    // Topic participants:
+    // Users who either sent or reacted to the messages in the topic.
+    const topic_messages = message_util.get_messages_in_topic(stream_id, topic);
+
+    for (const mssg of topic_messages) {
```

Comment:
let's call the local variable here `message`.

Reply:
Not needed as we are using the `topic_wildcard_mentioned` flag approach now.